ai_chatbots/consumers.py

```diff
@@ -314,6 +303,18 @@ async def handle(self, message: str):
         except (ValidationError, json.JSONDecodeError) as err:
             log.exception("Bad request")
             await self.send_error_response(400, err, cookies)
+        except AsyncThrottled as err:
+            log_msg = "User %s throttled on %s for %d seconds" % (
+                self.get_ident(),
+                self.__class__.__name__,
+                err.wait,
+            )
+            log.info(log_msg)
+            await self.start_response(thread_id=None, status=200, cookies=cookies)
+            await self.send_chunk(
+                f"You have reached the maximum number of chat requests.\
+                \nPlease try again in {format_seconds(err.wait)}."
+            )
```

Comment:
I waffled back and forth between:

a) sending an error response with status code 429, and the above message in the error details.  This would require additional front-end changes to notify the user about the rate limit being reached.
b) sending the rate limit info directly to the user as above as a normal response.

I'm fine with changing to (a) if you feel it is a better approach.